src/components/search/index.tsx

```diff
@@ -16,33 +18,36 @@ const formatSearch = (searchTerm: string): string => {
 };
 
 export const SearchInput: React.FC = () => {
+  const [names, setNames] = useState<string[] | []>([]);
   const handleChange = async (e: React.ChangeEvent<HTMLInputElement>) => {
     const searchTerm = e.target.value;
-    console.log('Search term: ', searchTerm);
     try {
       const search = formatSearch(searchTerm);
       const searchData = await getSearchResults(search);
-      console.log(searchData);
+      const productNames = searchData.map((product) => product.name);
+      setNames(productNames);
```

Comment:
I think you might have to pass not just the name, but the ID as well. At some point, we'll want to click on these items and fetch a product from the API. So the item needs to have access to the ID as well. You can pass a product object with id and name.